### PR TITLE
RotateServiceAccount documentation update.

### PR DESCRIPTION
**Pull request by dekiel**
**Description**

Changes proposed in this pull request:

Updated documentation to reflect migration to cloud-run service and adding of terraform config files.

### CODE REVIEW

**Comment by halamix2** [comment]
Is that placeholder needed?

**Comment by dekiel** [comment]
> Is that placeholder needed?

Yes, if the directory would not exist, a governance test would fail on broken link. I will build tf config in separate PR.

**Review by halamix2 — DISMISSED** [review]

File: development/secrets-rotator/cloud-run/rotate-service-account/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+1. Secret Manager sends a Pub/Sub message to `secret-manager-notifications` Pub/Sub topic.
+3. RotateServiceAccount checks if the value of the **eventType** attribute is set to `SECRET_ROTATE`; if not, it stops its execution.
+4. RotateServiceAccount checks if the value of the **type** label is set to `service-account`; if not, it stops its execution.
+5. RotateServiceAccount reads the name of the service account from the latest version of a secret.
+6. RotateServiceAccount generates a new key for the service account.
+7. RotateServiceAccount creates a new secret version in Secret Manger, containing the newly created service account key.
+
+## Cloud Run deployment
+
+RotateServiceAccount is deployed to cloud run applying Terraform config stored in [./terraform directory](../../terraform). Terraform apply is executed automatically on every PR changing Terraform .tf files belonging to the application.
```
Comment (IwonaLanger) [inline development/secrets-rotator/cloud-run/rotate-service-account/README.md:16]:
```suggestion
RotateServiceAccount is deployed to Cloud Run applying Terraform config stored in [./terraform directory](../../terraform). `terraform apply` is executed automatically on every PR changing Terraform .tf files belonging to the application. 
```

File: development/secrets-rotator/cloud-run/rotate-service-account/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+6. RotateServiceAccount generates a new key for the service account.
+7. RotateServiceAccount creates a new secret version in Secret Manger, containing the newly created service account key.
+
+## Cloud Run deployment
+
+RotateServiceAccount is deployed to cloud run applying Terraform config stored in [./terraform directory](../../terraform). Terraform apply is executed automatically on every PR changing Terraform .tf files belonging to the application. 
+
+## RotateServiceAccount usage
+
+To setup an automatic rotation for a Secret Manager secret follow these steps:
```
Comment (IwonaLanger) [inline development/secrets-rotator/cloud-run/rotate-service-account/README.md:20]:
```suggestion
To setup an automatic rotation for a Secret Manager secret, follow these steps:
```

File: development/secrets-rotator/cloud-run/rotate-service-account/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+1. Create a new secret in Secret Manager with the existing service account data.
+2. Add `type: service-account` label to the secret.
```
Comment (IwonaLanger) [inline development/secrets-rotator/cloud-run/rotate-service-account/README.md:22]:
```suggestion
2. Add the `type: service-account` label to the secret.
```

File: development/secrets-rotator/cloud-run/rotate-service-account/README.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+3. Set `secret-manager-notifications` as a secret Pub/Sub topic.
+4. Set up rotation period for the secret.
```
Comment (IwonaLanger) [inline development/secrets-rotator/cloud-run/rotate-service-account/README.md:24]:
```suggestion
4. Set up a rotation period for the secret.
```

**Review by IwonaLanger — APPROVED** [review]

**Comment by dekiel** [comment]
Dockerfile and tf config will be provided in separate PRs.
Dockerfile PR: #6977 
/override pull-build-image-rotate-service-account
/override pull-secrets-rotator-tf-plan

**Review by halamix2 — APPROVED** [review]

**Pull request merged**